sequencer/sequencesender.go

```diff
@@ -175,7 +175,7 @@ func (s *Sequencer) handleEstimateGasSendSequenceErr(
 
 	// while estimating gas a new block is not created and the POE SC may return
 	// an error regarding timestamp verification, this must be handled
-	if strings.Contains(err.Error(), errTimestampMustBeInsideRange) {
+	if errors.Is(err, ethman.ErrTimestampMustBeInsideRange) {
```

Comment:
```suggestion
	if errors.Is(err, etherman.ErrTimestampMustBeInsideRange) {
```

Reply:
this is not valid as it collides with the interface inside the package: https://github.com/0xPolygonHermez/zkevm-node/blob/develop/sequencer/interfaces.go#L32